_posts/2020-10-16-kaitai-struct-v0.9-released.md

```diff
@@ -99,12 +101,19 @@ fixes a lot of bugs and includes quite a few infrastructure improvements.
     Gradle plugin](https://github.com/valery1707/kaitai-gradle-plugin)
 * Infrastructure updates:
   * Unstable binary builds are available for all platforms after every CI build at Bintray ([#63](https://github.com/kaitai-io/kaitai_struct/issues/63))
-  * KSY language reference replaced with [documentation](https://doc.kaitai.io/ksy_diagram.html) generated from JSON schema
+  * KS language reference replaced with [documentation](https://doc.kaitai.io/ksy_diagram.html) generated from [the JSONSchema](https://github.com/kaitai-io/ksy_schema)
```

Comment:
It's called [**JSON Schema**](https://json-schema.org/), not ~JSONSchema~, ~JSONS chema~ or whatever:

```suggestion
  * KS language reference replaced with [documentation](https://doc.kaitai.io/ksy_diagram.html) generated from [the JSON Schema](https://github.com/kaitai-io/ksy_schema)
```